src/test/java/org/folio/rest/api/HoldingsStorageTest.java

```diff
@@ -791,6 +791,212 @@ public void holdingsCallNumberDoesNotUpdateItemWithItemLevelCallNumber()
     assertThat(updatedItemFromGet.getJsonObject("effectiveCallNumberComponents").getString("callNumber"), is("itemLevelCallNumber"));
   }
 
+  @Test
+  public void updatingHoldingsUpdatesItemEffectiveCallNumberSuffix()
```

Comment:
Could another test be added to check that an item under a different holdings does not get updated?

Reply:
I've addressed it #358